docs/releases/4.1.txt

```diff
@@ -275,6 +275,11 @@ Management Commands
 * The new :djadmin:`optimizemigration` command allows optimizing operations for
   a migration.
 
+* :djadmin:`sqlmigrate` now outputs ``(no-op)`` for migration operations that
+  do not execute any SQL. Output for operations that aren’t representable as
+  SQL, such as :class:`~django.db.migrations.operations.RunPython`, has changed
+  to ``THIS OPERATION CANNOT BE WRITTEN AS SQL``.
```

Comment:
I know it's a cosmetic change that wouldn't normally deserve a release note, but I worry that someone out there has a script that runs `sqlmigrate` and greps for the old string `MIGRATION NOW PERFORMS OPERATION THAT CANNOT BE WRITTEN AS`

Reply:
I'd chop it. We cannot document all wording changes, this is an implementation details that Django doesn't guarantee.

Reply:
ok fair, will drop